roles/ansible_tower/tasks/main.yaml

```diff
@@ -32,3 +31,4 @@
 - name: Set (httpd_can_network_connect_db) flag on and keep it persistent across reboots
   seboolean: name=httpd_can_network_connect_db state=yes persistent=yes
 
+- include: ../../ansible_tower_cli/tasks/main.yml
```

Comment:
While this works, I'd either just add this role to the tower config playbook, or add a requirement of the ansible_tower_cli role to this role's metadata.